### PR TITLE
chore(release): 6.8.0

### PR DESCRIPTION
**Pull request by nellyk**

### CODE REVIEW

**Comment by JamesSingleton** [comment]
This shouldn't be changing everything right? Only the generator was changed with @infoxicator PRs.

File: packages/one-app-runner/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,34 @@
...
+
+* **one-app-runner:** add ([#34](https://github.com/americanexpress/one-app-cli/issues/34)) ([f3c6755](https://github.com/americanexpress/one-app-cli/commit/f3c67551ec9458f30ddf640666c69f3e673c0784))
+* **one-app-runner:** create network and env file ([#78](https://github.com/americanexpress/one-app-cli/issues/78)) ([d8ac5ec](https://github.com/americanexpress/one-app-cli/commit/d8ac5ec8a36413217d942e9c5d611c4008b3f346))
+* **runner:** configuration for browser tests ([#67](https://github.com/americanexpress/one-app-cli/issues/67)) ([4ae5eab](https://github.com/americanexpress/one-app-cli/commit/4ae5eabc4857e96ed39ed8708054f10c151891d6))
+* **runner:** require modules if moduleMapUrl is missing ([#48](https://github.com/americanexpress/one-app-cli/issues/48)) ([992af08](https://github.com/americanexpress/one-app-cli/commit/992af08a5dde7d69c6ee3578883c004c5f4d875c))
+
+
+### Reverts
+
+* Revert "chore(release): bump to v6.5.0 (#62)" ([5f708de](https://github.com/americanexpress/one-app-cli/commit/5f708de11f30163687f3184adb4d57ccab46649c)), closes [#62](https://github.com/americanexpress/one-app-cli/issues/62)
```
Comment (JAdshead) [inline packages/one-app-runner/CHANGELOG.md:28]: this might be confusing
```suggestion
```

File: packages/generator-one-app-module/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,48 @@
...
+* **deps:** update generator deps for react-intl and parrot-generator ([ac24226](https://github.com/americanexpress/one-app-cli/commit/ac24226ee1de6ac2f5b02c30050fc8a052c40f35))
+* **generator:** csp report-uri to ONE_CLIENT_CSP_REPORTING_URL ([#50](https://github.com/americanexpress/one-app-cli/issues/50)) ([9ae5dc9](https://github.com/americanexpress/one-app-cli/commit/9ae5dc9fb0d63dc666f7c386fec732e12249435d))
+* **generator:** update base-module template ([#25](https://github.com/americanexpress/one-app-cli/issues/25)) ([1a0af74](https://github.com/americanexpress/one-app-cli/commit/1a0af748f94790ceae7b2a87fc827be2d549cf6c))
+
+
+### Reverts
+
+* Revert "chore(release): bump to v6.5.0 (#62)" ([5f708de](https://github.com/americanexpress/one-app-cli/commit/5f708de11f30163687f3184adb4d57ccab46649c)), closes [#62](https://github.com/americanexpress/one-app-cli/issues/62)
+
+
```
Comment (JAdshead) [inline packages/generator-one-app-module/CHANGELOG.md:35]:
```suggestion
```

File: packages/eslint-plugin-one-app/CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,46 @@
...
+### Bug Fixes
+
+* **css-loader:** upgraded to latest css-loader api as of 3.4.2 ([74cad69](https://github.com/americanexpress/one-app-cli/commit/74cad69fcbe84eeba7a02b009821e6f7a2db62f2))
+* **externals:** resolver did not recognize complex browser fields ([#66](https://github.com/americanexpress/one-app-cli/issues/66)) ([d35d03c](https://github.com/americanexpress/one-app-cli/commit/d35d03c858a1361a1f214ff3f99b194d2aede521))
+
+
+### Reverts
+
+* Revert "chore(release): bump to v6.5.0 (#62)" ([5f708de](https://github.com/americanexpress/one-app-cli/commit/5f708de11f30163687f3184adb4d57ccab46649c)), closes [#62](https://github.com/americanexpress/one-app-cli/issues/62)
+
```
Comment (JAdshead) [inline packages/eslint-plugin-one-app/CHANGELOG.md:22]:
```suggestion
```

**Comment by nellyk** [comment]
Closing to create a new PR with the released changes

**Pull request closed without merging**